docs/tutorials/ibc.md

```diff
@@ -0,0 +1,95 @@
+# IBC
+
+This is more of a cheat sheet than a tutorial at this stage.
+
+## Creating an ibc client
+
+```shell
+gaiad tx ibc client create ibc_config/tendermint_client_state.json ibc_config/tendermint_consensus_state.json --from bob --keyring-backend=test --chain-id=test-chain
+```
+
+Which gives the following app hash in the go implementation of gaia (this isn't deterministic):
+
+app_hash=????
+
+```shell
+gaia-rs tx local alice ibc client create ibc_config/tendermint_client_state.json ibc_config/tendermint_consensus_state.json
+```
+
+```shell
+cargo run -- tx local alice ibc client create docs/tutorials/ibc_config/gaia_rs_client_state.json docs/tutorials/ibc_config/gaia_rs_consensus_state.json
+```
+
+Check if it was created:
+
+```shell
+gaiad query ibc client states --output=json
+```
+
+```shell
+cargo run -- query ibc client states
+```
+
+## Relaying with hermes
+
+1. Add chain data to hermes config in `~/.hermes/config.toml`:
+
+```toml
+[[chains]]
+id = 'test-chain'
+rpc_addr = 'http://localhost:26657'
+grpc_addr = 'http://localhost:9090'
+websocket_addr = 'ws://localhost:26557/websocket'
+rpc_timeout = '10s'
+account_prefix = 'atom'
+key_name = 'testkey'
+store_prefix = 'ibc'
+max_gas = 2000000
+gas_price = { price = 0.001, denom = 'stake' }
+gas_adjustment = 0.1
+clock_drift = '5s'
+trusting_period = '14days'
+trust_threshold = { numerator = '1', denominator = '3' }
+```
+
+2. Query packets
+
+```shell
+hermes query packet unreceived-packets test-chain transfer channel-7
+```
+
+```shell
+hermes query packet acks test-chain transfer channel-7
+```
+
+response (failed):
+
+```txt
+Got query request to: store/ibc/key
+```
+
+### Opening a connection
+
+This requires keys:
+
+```shell
+hermes keys restore test-chain --mnemonic "race draft rival universe maid cheese steel logic crowd fork comic easy truth drift tomorrow eye buddy head time cash swing swift midnight borrow"
```

Comment:
Is it expected to know what is `hermes`? I suppose that in future there will be more instructions on, but if we depend on specific version (as we do with tendermint) than you need to leave link to this.

Reply:
Agreed. The plan is to turn this into documentation at some point. Right now it's more of a cheat sheet.